2-deprecate-big-bang/README.md

```diff
@@ -0,0 +1,285 @@
+
+
+### Goals
+
+- Remove all references to Big Bang besides the code paths following `zarf dev generate big-bang`.
+- Workflows on Pull Requests (PRs) should not pull from Iron Bank.
+
+## Proposal
+
+The proposed solution is creating a new Zarf Command Line interface (CLI) command `zarf dev generate big-bang`. The keys `valuesFiles`, `skipFlux`, and `repo` will become CLI arguments to the command. The `fluxPatchFiles` key will not have an equivalent, however, users will be able to edit their Flux component to deploy with patch files after the generate command has run. Below is the helper text for the command: 
+
+```bash
+Generates a zarf.yaml file and the associated manifests necessary to create a Zarf package that deploys Big Bang 
+Usage:
+  zarf dev generate big-bang [ Version ] [flags]
+
+Examples:
+zarf dev generate big bang --version 2.3.4 –skip-flux=false --values-file=istio-values.yaml,
+
+Flags:
+  --skip-flux bool  Whether or not to create a flux component (default false)
+  --repo string   	Override repo to pull Big Bang from instead of Repo One.
+  --values-files    A comma separated list of values files to pass to the Big Bang component
+
+```
+
+This command will create a zarf.yaml file with a component that fully deploys Big Bang for the provided version. Zarf will generate manifests and place them in a manifests folder. Any files submitted with the `valuesFiles` flag will be added to the manifests as well. The component will follow the following structure: 
+
+```
+- Manifests
+  - Flux Git Source Manifest
+  - Flux Helm Release Manifest
+  - Zarf custom values file
+  - User submitted custom values files
+- Images
+  - All the Big Bang images
+- Git Repositories
+  - All the Big Bang repos
+- Health checks
+  - A health check for each Helm Release
+- OnRemove actions
+  - Removing each Helm Release
+```
+
+If a zarf.yaml file already exists `zarf dev generate big-bang` will instead generate a file called `bigbang-zarf.yaml`. This will prevent Zarf from overwriting any existing Zarf files while providing a convenient way to copy and paste images and repos from one file to the other.
+
+### User Stories (Optional)
+
+#### Story 1
+
+A user wants to deploy Big Bang with Zarf. They run `zarf dev generate big-bang 2.34.0` without an existing `zarf.yaml` and get the necessary manifests and zarf.yaml so that they are ready to run `zarf package create`.
+
+#### Story 2
+
+An existing deployer of Big Bang with Zarf wants to update to a new version of Big Bang. They already have a `zarf.yaml` that deploys Big Bang alongside other components - shown below. They run `zarf dev generate big-bang 2.34.0` which creates a `bigbang-zarf.yaml` so that their existing `zarf.yaml` is not overridden. The user then copies and and pastes the images and repos from the `bigbang-zarf.yaml` to their existing `zarf.yaml`.
```

Comment:
```suggestion
An existing deployer of Big Bang with Zarf wants to update to a new version of Big Bang. They already have a `zarf.yaml` that deploys Big Bang alongside other components - shown below. They run `zarf dev generate big-bang 2.34.0` which creates a `bigbang-zarf.yaml` so that their existing `zarf.yaml` is not overridden. The user then copies and pastes the images and repos from the `bigbang-zarf.yaml` to their existing `zarf.yaml`.
```